src/dom_components/view/ComponentTextView.js

```diff
@@ -91,6 +91,7 @@ module.exports = ComponentView.extend({
 
     this.rteEnabled = 0;
     this.toggleEvents();
+    this.render();
```

Comment:
I saw your comment above about "double content" (lines 67-68)...seems like it's related, so maybe there's a place to fix this?

Reply:
Unfortunately, this breaks stuff, try for example to remove a word and then execute undo and redo, this will duplicate the content. The issue is with `model.set('content', '')` (line 85), it doesn't trigger the change as the `content` is already empty so it doesn't reset its content. 
For now the only fix I see it's something like this `model.set('content', Math.random()).set('content', '')`